frontend/src/metabase/components/ItemsTable/BaseItemTableRow.tsx

```diff
@@ -65,7 +65,7 @@ export const TableRow = ({
       onCopy={onCopy}
       onMove={onMove}
       onToggleSelected={onToggleSelected}
-      showActionMenu={showActionMenu}
+      visibleColumnsMap={visibleColumnsMap}
```

Comment:
nit: I noticed that the other component uses `visibleColumns` while this prop uses `visibleColumnsMap`, perhaps we want to use the same prop name?

Reply:
@deniskaber I totally missed that the type differs there, thanks!